### PR TITLE
fixed bug from audit report

### PR DESCRIPTION
**Pull request by parth-15**
Fixed bugs highlighted in audit reports.

### CODE REVIEW

File: src/facets/BoostFacet.sol [hunk 1]
```diff
@@ -39,11 +40,18 @@ contract BoostFacet {
         if (isStratosphereMember) {
             _boostFee = s.boostLevelToFee[boostLevel];
         }
+        _userData.lastBoostClaimTimestamp = block.timestamp;
+        uint256 _boostPointsAmount;
```
Comment (mejiasd3v) [inline src/facets/BoostFacet.sol:44]: this variable is not storing points amount but the boost percentage, how about we rename it `boostPercent`?

File: src/facets/DiamondManagerFacet.sol [hunk 2]
```diff
@@ -86,6 +90,13 @@ contract DiamondManagerFacet {
         if (receivers.length != proportion.length) {
             revert DiamondManagerFacet__Invalid_Input();
         }
+        uint256 totalProportion = 0;
+        for (uint256 i = 0; i < proportion.length; i++) {
```
Comment (mejiasd3v) [inline src/facets/DiamondManagerFacet.sol:94]: no need to initialise i to 0, also to keep it consistent I'd rename proportions to shares

File: src/facets/DiamondManagerFacet.sol [hunk 3]
```diff
@@ -95,6 +106,13 @@ contract DiamondManagerFacet {
...
```
Comment (mejiasd3v) [inline src/facets/DiamondManagerFacet.sol:110]: same here no need for i = 0

File: src/facets/DiamondManagerFacet.sol [hunk 4]
```diff
@@ -104,6 +122,13 @@ contract DiamondManagerFacet {
...
```
Comment (mejiasd3v) [inline src/facets/DiamondManagerFacet.sol:126]: same here no need for i = 0

File: src/facets/DiamondManagerFacet.sol [hunk 5]
```diff
@@ -113,6 +138,13 @@ contract DiamondManagerFacet {
...
```
Comment (mejiasd3v) [inline src/facets/DiamondManagerFacet.sol:142]: same here no need for i = 0

File: src/facets/DiamondManagerFacet.sol
Comment (mejiasd3v) [inline src/facets/DiamondManagerFacet.sol:1]: I see we are using a lot of times 10_000, can we move this somewhere where we can reuse it from? like creating a constant in LPercentages such as `PRECISION`

File: src/upgradeInitializers/DiamondInit.sol [hunk 6]
```diff
@@ -62,18 +62,18 @@ contract DiamondInit {
...
-        s.unlockFeeDiscountForStratosphereMembers[1] = 500;
-        s.unlockFeeDiscountForStratosphereMembers[2] = 550;
-        s.unlockFeeDiscountForStratosphereMembers[3] = 650;
-        s.depositDiscountForStratosphereMembers[1] = 500;
-        s.depositDiscountForStratosphereMembers[2] = 550;
-        s.depositDiscountForStratosphereMembers[3] = 650;
-        s.restakeDiscountForStratosphereMembers[1] = 500;
-        s.restakeDiscountForStratosphereMembers[2] = 550;
-        s.restakeDiscountForStratosphereMembers[3] = 650;
+        s.unlockTimestampDiscountForStratosphereMembers[2] = 650;
```
Comment (mejiasd3v) [inline src/upgradeInitializers/DiamondInit.sol:67]: this seems to be overriding each other

**Review by royvardhan — APPROVED** [review]

**Review by mejiasd3v — APPROVED** [review]

**Pull request merged**